gapic/src/main/com/google/gapid/widgets/LinkifiedTree.java

```diff
@@ -276,7 +276,7 @@ protected T getElement(Widget item) {
 
     @Override
     public Object[] getChildren(Object parent) {
-      return getChildNodes(cast(parent));
+      return !hasChildren(parent) ? null : getChildNodes(cast(parent));
```

Comment:
I would avoid the negative test (!):

```suggestion
      return hasChildren(parent) ? getChildNodes(cast(parent)) : null;
```